mocks/ca.go

```diff
@@ -44,9 +44,7 @@ func (ca *MockCA) IssueCertificateForPrecertificate(ctx context.Context, req *ca
 		RegistrationID: 1,
 		Serial:         "mock",
 		Digest:         "mock",
-		IssuedNS:       now.UnixNano(),
 		Issued:         timestamppb.New(now),
-		ExpiresNS:      expires.UnixNano(),
 		Expires:        timestamppb.New(expires),
```

Comment:
Similar to my comment above about inlining `now`, the value of `expires` can also be inlined here. The same is true for variables like `created` and `issuedTime` in the next file and throughout.